config/dev.exs

```diff
@@ -76,4 +76,4 @@ config :phoenix, :stacktrace_depth, 20
 config :phoenix, :plug_init_mode, :runtime
 
 # add pre-commit command see #20
-config :pre_commit, commands: ["test", "coveralls"]
+config :pre_commit, commands: ["format", "test", "coveralls"]
```

Comment:
thanks for adding this. 👍

Reply:
In fact I'm not sure adding the format command on pre-commit is that useful as any changes produce by `format` won't be added to the commit I think.
The best way is to configure our editor to run `mix format` when a file is saved